x-pack/plugins/security_solution/cypress/tasks/common.ts

```diff
@@ -179,6 +179,23 @@ export const deleteCases = () => {
   });
 };
 
+export const deleteConnectors = () => {
+  const kibanaIndexUrl = `${Cypress.env('ELASTICSEARCH_URL')}/.kibana_\*`;
+  cy.request('POST', `${kibanaIndexUrl}/_delete_by_query?conflicts=proceed`, {
+    query: {
+      bool: {
+        filter: [
+          {
+            match: {
+              type: 'action',
+            },
+          },
+        ],
+      },
+    },
+  });
+};
```

Comment:
It would be great to extract all API calls from this `common` file to the corresponding files under `x-pack/plugins/security_solution/cypress/tasks/api_calls`